src/test/java/railway/RegisterTest.java

```diff
@@ -12,17 +12,17 @@ public class RegisterTest extends BaseTest {
     public HomePage homePage = new HomePage();
 
     @Test
-    public void TC() throws InterruptedException {
+    public void TC00() throws InterruptedException {
```

Comment:
1. What is this mean "TC00"?
2. Should not use "InterruptedException" in a test case

Reply:
Last week, the homework create a test case "User registers successfully then login with this account". It's not have in Set1 Testcase. So I set it "TC00"